documentation/API-design-guidelines.md

```diff
@@ -927,7 +928,7 @@ And according to the filtering based on string and enums data, being searched fo
 **Additional rules**:
 - The operator "`&`" is evaluated as an AND between different attributes.
 - A Query Param (attribute) can contain one or n values separated by "`,`".
-- For operations on numeric, date or enumerated fields, the parameters with the suffixes `.(gte|gt|lte|lt)$` need to be defined, which should be used as comparators for “greater—equal to, greater than, smaller—equal to, smaller than” respectively. Only the parameters needed for given field should be defined e.g., with `.gte` and `.lte` suffixes only.
+- For operations on numeric, date or enumerated fields, the parameters with the suffixes `.(gte|gt|lte|lt)$` need to be defined, which should be used as comparators for “greater—equal to, greater than, smaller—equal to, smaller than” ly. Only the parameters needed for given field should be defined e.g., with `.gte` and `.lte` suffixes only.
```

Comment:
typo: only "ly" remains for respectively

Reply:
Corrected in https://github.com/camaraproject/Commonalities/pull/333/commits/7c05167fd9bf29c45a2f8488025d4063a17e7b87